src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -596,6 +599,14 @@ private bool HasCallerMemberNameAttribute
             }
         }
 
+        private bool HasCallerArgumentExpressionAttribute
```

Comment:
>HasCallerArgumentExpressionAttribute [](start = 21, length = 36)

This property feels unnecessary. Also, it feels like we should be able to avoid looking for the attribute twice. First in this property and then to extract parameter name from it.

Reply:
@AlekseyTs It feels much more simple if I didn't follow the existing pattern for other caller info, i.e, not even touch PackedFlags.